app/main/helpers/validation.py

```diff
@@ -62,9 +62,6 @@ def get_error_message(self, question_id, message_key):
         return default_messages.get(
             message_key, 'There was a problem with the answer to this question')
 
```

Comment:
Are you suggesting keep it as documentation? That `NotImplementedError` can never raise. 
```
>>> class Foo(object):
	def bar(self):
		print 'baz'
	def bar(self):
		print 'quux'

		
>>> Foo().bar()
quux
>>> class InheritFoo(Foo):
	pass

>>> InheritFoo().bar()
quux
>>> 
```